examples/jms-broker/README.md

```diff
@@ -0,0 +1,137 @@
+
+NB: In order to adapt the configuration to the execution context, the expression `${env.BROKER_HOST}` 
+is set in the producer and mdb-consumer socket-binding `host` attribute value. The env variable is set in the WildFly Helm chart yaml file.
+
+## Extra content
+Extra content packaged inside the provisioned servers
+
+* None
+
+# Openshift build and deployment
+Technologies required to build and deploy this example
+
+* Helm chart for WildFly `wildfly/wildfly`. Minimal version `2.0.0`.
+
+# WildFly image API
+Environment variables from the [WildFly image API](https://github.com/wildfly/wildfly-cekit-modules/blob/main/jboss/container/wildfly/run/api/module.yaml) that must be set in the OpenShift deployment environment
+
+* None
+
+# WildFly cloud feature-pack added features
+
+* The remote connection factory is configured with `reconnect-attempts=-1`, allowing the clients to reconnect when the broker is down. 
+
+# Pre-requisites
+
+* You are logged into an OpenShift cluster and have `oc` command in your path
+
+* You have installed Helm. Please refer to [Installing Helm page](https://helm.sh/docs/intro/install/) to install Helm in your environment
+
+* You have installed the repository for the Helm charts for WildFly
+
+ ```
+helm repo add wildfly https://docs.wildfly.org/wildfly-charts/
+```
+----
+**NOTE**
+
+If you have already installed the Helm Charts for WildFly, make sure to update your repository to the latest version.
+
+```
+helm repo update
+```
+----
+
+# Example steps
+
+1. Create the persistent volume.
+
+```
+oc create -f broker/jms-volume-claim.yaml
+```
+
+2. Deploy the broker using WildFly Helm charts
+
+```
+helm install jms-broker -f broker/helm.yaml wildfly/wildfly
+```
+
+3. Deploy the producer using WildFly Helm charts
+
+```
+helm install jms-producer -f producer/helm.yaml wildfly/wildfly
+```
+
+4. Deploy the consumer using WildFly Helm charts
+
+```
+helm install mdb-consumer -f mdb-consumer/helm.yaml wildfly/wildfly
+```
+
+Wait for the pods to be ready. It can takes some minutes for the images to be built and deployed.
+You can check the status of the pods by calling: `oc get pods -w`.
+
+
+5. Produces messages
+
+`curl https://$(oc get route jms-producer --template='{{ .spec.host }}')/HelloWorldMDBServletClient`
+
+Check the logs of the mdb-consumer, you will see the traces of the received messages.
+
+# Broker failure scenario, messages are persisted
+
+1. Scale down the mdb-consumer
+
+`oc scale --replicas=0 deployments mdb-consumer`
+
+2. Produces messages
+
+`curl https://$(oc get route jms-producer --template='{{ .spec.host }}')/HelloWorldMDBServletClient`
+
+These messages are persisted, waiting for a consumer to come online.
+
+3. Scale down the broker, emulates a failure.
+
+`oc scale --replicas=0 deployments jms-broker`
+
+4. Scale up the broker
+
+`oc scale --replicas=1 deployments jms-broker`
+
```

Comment:
It would be good here to repeat that you cannot scale beyond one.

Reply:
Text added there too.